client/state/data-layer/wpcom/read/teams/index.js

```diff
@@ -3,9 +3,20 @@
  */
 import { READER_TEAMS_REQUEST, READER_TEAMS_RECEIVE } from 'state/action-types';
 import wpcom from 'lib/wp';
+import inflight from 'lib/inflight';
 
 export function handleTeamsRequest( store, action, next ) {
-	wpcom.req.get( '/read/teams', { apiVersion: '1.2' } )
+	const dedupeKey = READER_TEAMS_REQUEST;
+	if ( inflight.requestInflight( dedupeKey ) ) {
```

Comment:
we should also check if we've already loaded the teams and ignore the request if so. or maybe remember _when_ we loaded the teams and reload if it's been a bit.